.github/workflows/tests.yml

```
@@ -18,6 +18,13 @@ jobs:
         extensions: 'hpp,cpp'
         clangFormatVersion: 12
 
+  reuse:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: REUSE Compliance Check
```

I'd make it a step of the lint job

```suggestion
    - uses: actions/checkout@v4
      name: REUSE Compliance Check
```

---

```suggestion
    - name: REUSE Compliance Check
      if: always()
```
That way, it will

---

Ok why not, why is it better ?

---

We don't spin up a whole new VM for a 1s job. Should slightly decrease pressure on runners